linera-service/template/rust-toolchain.template

```diff
@@ -0,0 +1 @@
+1.69.0
```

Comment:
Should this be a symbolic link to `/rust-toolchain` ?

Reply:
better use https://github.com/linera-io/linera-protocol/blob/main/rust-toolchain

Reply:
@firedpeanut No, we need a file or a symlink at `linera-service/template/rust-toolchain.template`, otherwise `cargo publish` will fail to include the file in the crate.

Reply:
OK. rollback push